pkg/server/handlers.go

```diff
@@ -196,23 +262,24 @@ func createDeployment(w http.ResponseWriter, r *http.Request) {
 		tempPTS.Annotations = make(map[string]string)
 	}
 
-	if tempJSON.PrivateHosts != nil {
-		tempPTS.Annotations["privateHosts"] = *tempJSON.PrivateHosts
-	}
-
-	if tempJSON.PublicHosts != nil {
-		tempPTS.Annotations["publicHosts"] = *tempJSON.PublicHosts
-	}
+	tempPTS.Annotations["edge/paths"] = composePathsJSON(tempJSON.Paths)
```

Comment:
I agree it would be great to have a better system for configuring these variables. I'll look into viper a little bit to see if that can solve our problems as that is what `30x/dispatcher` is using.